xonsh/environ.py

```diff
@@ -21,6 +21,11 @@ def current_branch(cwd=None, pad=True):
     bust should be extended in the future.
     """
     branch = None
+
+    if platform.system() == 'Windows':
```

Comment:
For consistency and speed, it would be nice to have (a) global variable(s) in tools.py that is something like:

```python
ON_WINDOWS = (platform.system() == 'Windows')
ON_POSIX = (os.name == 'posix')
```

This saves on memory and speed and decreases the likelihood of annoying typos.